### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tmigot**
Correct links to NLPModels.jl and OptimizationProblems.jl

### CODE REVIEW

**Comment by dpo** [comment]
The link is in square brackets and the text to display in parentheses. It was correct, wasn't it?

**Comment by tmigot** [comment]
Right now, when we click on the link, for instance NLPModels.jl, we go to:
https://github.com/JuliaSmoothOptimizers/NLSProblems.jl/blob/master/NLPModels.jl

**Comment by abelsiqueira** [comment]
It actually inverted, it should be [title](link).

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 
 This package provides some optimization problems using the
-[https://github.com/JuliaSmoothOptimizers/NLPModels.jl](NLPModels.jl)
+[https://github.com/JuliaSmoothOptimizers/NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl)
```
Comment (dpo) [inline README.md]: Should be `[NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl)`

**Comment by dpo** [comment]
Thank you!

**Pull request merged**